src/@next/Tag/TagStyle.ts

```diff
@@ -1,18 +1,66 @@
 import styled from 'styled-components';
 import { Breakpoints } from '..';
-import { borderRadius4 } from '../utilities/borderRadius';
-import { Blue } from '../utilities/colors';
+import {
+  borderRadius2,
+  borderRadius4,
+  borderRadiusHalf,
+} from '../utilities/borderRadius';
+import { Blue, Neutral } from '../utilities/colors';
 import { space4, space8 } from '../utilities/spacing';
-import { TagProps } from './Tag';
+import { TagContentProps, TagProps, TagRemoveContainerProps } from './Tag';
+
+export const TagContentStyle = styled.span<TagContentProps>`
+  padding: ${space4} ${space8};
+
+  &[data-removeable='true'] {
+    padding-right: 0;
+  }
+`;
+
+export const TagRemoveContainerStyle = styled.div<TagRemoveContainerProps>`
+  padding: ${space4};
+  display: flex;
+`;
+
+export const TagIconWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &:focus-visible {
+    margin: -2px;
+    outline: none;
+    border: 2px solid ${Blue.S100};
+    border-radius: ${borderRadius4};
+  }
+`;
 
 export const TagStyle = styled.div<TagProps>`
   display: inline-flex;
+  align-items: center;
   background-color: ${Blue.S08};
-  width: fit-content;
-  padding: ${space4} ${space8};
+
   border-radius: ${borderRadius4};
+  width: fit-content;
 
   @media (max-width: ${Breakpoints.large}) {
     font-size: 12px;
   }
+
+  & svg {
+    padding: 0;
+  }
+
+  & svg:hover {
+    cursor: pointer;
+    background-color: ${Neutral.B95};
+    border-radius: ${borderRadiusHalf};
+  }
+
+  & svg:active {
```

Comment:
can you include active state in the test?

Reply:
that's the challenge i have, i am not sure how to do long press in playwright and then take screenshot

Reply:
added the visual test for pressed state